test/zdtm/lib/ns.c

```diff
@@ -51,6 +51,15 @@ static int prepare_mntns(void)
 		return -1;
 	}
 
+	dev_path = getenv("ZDTM_DEV");
+	if (dev_path) {
+		snprintf(path, sizeof(path), "%s/dev", root);
+		if (mount(dev_path, path, NULL, MS_BIND | MS_REC, NULL)) {
```

Comment:
As we've just created ZDTM_DEV mount we know for sure that it does not have children mounts, maybe we can remove MS_REC here?